### PR TITLE
Host in self signed cert

### LINKED ISSUE #42: Self generated cert for the kube-rbac-proxy only generates CN with timestamp
On running kube-rbac-proxy with only secure-listen-address and upstream, the generated certificate has CN of the value (only with Unix timestamp. The hostname is missing in the CN): 
subject=/CN=@1554404348
issuer=/CN=@1554404348

Expected:
CommonName: fmt.Sprintf("%s-ca@%d", host, time.Now().Unix())

- https://github.com/brancz/kube-rbac-proxy/blob/master/vendor/k8s.io/client-go/util/cert/cert.go#L184

- https://github.com/brancz/kube-rbac-proxy/blob/master/main.go#L222

### PR DESCRIPTION
**Pull request by brancz**
Fixes #42 

@s-urbaniak @squat @metalmatze @paulfantom

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,6 +5,7 @@ go 1.12
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
 	github.com/beorn7/perks v0.0.0-20160229213445-3ac7bf7a47d1 // indirect
+	github.com/campoy/embedmd v1.0.0 // indirect
```
Comment (squat) [inline go.mod]: how did this get in here?

**Comment by metalmatze** [comment]
My guess on embedmd, it wasn't simply executed in a previous run, thus not added to those go.mod files. The README.md uses embedmd, so it should be a dep of this project.
LGTM

**Comment by squat** [comment]
hmm but now it is in the mod file but still not in the vendor dir

**Comment by brancz** [comment]
Note that this is based on top of https://github.com/brancz/kube-rbac-proxy/pull/41, so let's first concentrate on getting that merged.

File: go.mod (hunk 1 above)
Comment (paulfantom) [reply go.mod]: `go get ...` from Makefile.

**Pull request merged**